### DISCUSSION TITLE
Welcome to Stickerify Bot Discussions!

### DISCUSSION THREAD

**Discussion by rob93c** [Stickerifier/Stickerify#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sorrow8** [comment]
How can I get webp format?

**Reply by rob93c** [reply]
Hi @H07633R, can you please describe your request?
What are you trying to send the bot and what's the outcome you would expect?

**Reply by sorrow8** [reply]
Hey Rob. Thanks for quick reply. I was trying to create a sticker format file with my .GIF using your bot. I got a reply from your bot, in screenshot 1. Then uploaded it to telegram Sticker creator bot, screenshot 2. And it looks like it only accepts .WEBP

![image](https://github.com/Stickerifier/Stickerify/assets/90210052/0a1ca87b-494a-45de-b66b-e8d53ec2ff05)

![image](https://github.com/Stickerifier/Stickerify/assets/90210052/294b714d-f369-479f-9697-da18ee0f6c77)

**Reply by rob93c** [reply]
Oh, I get the confusion! You are trying to add a video sticker to a static sticker pack (they allow only images and not videos), if you create the pack using the command `/newvideo` it should allow you to add your new video sticker.

Please let me know if anything goes wrong following these steps

**Reply by sorrow8** [reply]
Ouch. What a silly mistake of me. Your advice helped me. Thank you.
